Formula/ffmpeg.rb

```diff
@@ -67,9 +69,11 @@ def install
       --enable-libass
       --enable-libopencore-amrnb
       --enable-libopencore-amrwb
+      --enable-libopenjpeg
```

Comment:
Could you also add a `depends_on "openjpeg"`?

Reply:
Oops! Sorry about that!